lib/slimmer/processors/search_placeholder_setter.rb

```diff
@@ -0,0 +1,18 @@
+module Slimmer::Processors
+  class SearchPlaceholderSetter
+    def initialize(headers)
+      @headers = headers
+    end
+
+    def filter(_source, template)
+      if placeholder && template.at_css('form#search')
+        template.at_css('form#search label').content = placeholder
+        template.at_css('#site-search-text')['title'] = placeholder
```

Comment:
The potential assignment to nil if `#site-search-text` doesn't exist made me look twice, but I can't see a reason why it would be..